pkg/inspect/inspect.go

```diff
@@ -14,14 +15,25 @@ const (
 	REALITY_SOURCE
 )
 
+type LaunchableVersion struct {
+	Location string                    `json:"location,omitempty"`
```

Comment:
To make sure I understand: this field, and thus this whole type will is only here to support a diversity of manifest versions. If that's the case, I'm surprised that this type is only useful out here at the edge. It might be better to encapsulate this sort of thing in either: `launch.LaunchableVersion` or, if we want that type to stay simple, an adapter in the `manifests` package.